packages/commerce-sdk-react/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## v4.0.0-dev (Jun 21, 2024)
+## v3.1.1-extensibility-preview.0 (Dec 02, 2024)
```

Comment:
Why did we bump both the minor and bug versions here? We could possibly get away with not bumping the version at all.